src/Service.Host/client/src/components/Navbar.js

```diff
@@ -1,58 +1,163 @@
+import AccountCircle from '@material-ui/icons/AccountCircle';
+import Panel from './Panel';
 
-const styles = {
+const styles = theme => ({
     root: {
-        width: '100%',
         position: 'absolute',
-        top: 0
-    },
-    tab: {
-        minWidth: 20
+        width: '100%',
+        top: 0,
+        zIndex: 10
     },
     tabLabel: {
         fontSize: '12px'
+    },
+    panel: {
+        position: 'relative'
+    },
+    menuButton: {
+        marginLeft: -12,
+        marginRight: 20
+    },
+    fullHeight: {
+        ...theme.mixins.toolbar,
+        minWidth: '100px'
+    },
+    toolbar: {
+        paddingLeft: 0,
+        paddingRight: 0
+    },
+    tabs: {
+        ...theme.mixins.toolbar
+    },
+    container: {
+        width: '100%'
     }
-};
+});
 
-function Navbar({ classes, menu, loading, history }) {
+function Navbar({ classes, menu, loading, username }) {
     const [selected, setSelected] = useState(false);
-
-    const handleChange = (event, value) => {
+    const [anchorEl, setAnchorEl] = useState();
+    if (menu) {
         const menuIds = menu.map(item => item.id);
-        if (value || value === 0) {
-            history.push(`/${menuIds[value]}`);
-            setSelected(value);
-        }
-    };
+
+        const handleClick = event => {
+            setAnchorEl(event.currentTarget);
+        };
+        const handleClose = () => {
+            setAnchorEl();
+        };
+        const handleSignOut = () => {
+            window.location.assign('https://www.linn.co.uk/signout');
+        };
+
+        return (
+            <ClickAwayListener onClickAway={() => setSelected(false)}>
+                <div className={classes.root}>
+                    {menu && !loading ? (
+                        <AppBar position="static" color="default">
+                            <Toolbar classes={{ gutters: classes.toolbar }}>
+                                <Grid
+                                    container
+                                    alignItems="center"
+                                    justify="space-between"
+                                    spacing={24}
+                                    classes={{ container: classes.container }}
+                                >
+                                    <Fragment>
+                                        <Grid item xs={11}>
+                                            <Tabs
+                                                classes={{
+                                                    root: classes.tabs
+                                                }}
+                                                value={selected}
+                                                onChange={(event, value) => setSelected(value)}
+                                                scrollButtons="auto"
+                                                variant="scrollable"
+                                                indicatorColor="primary"
```

Comment:
where do we set the primary color of the app?

Reply:
I don't think we ever do - it just defaults to the default MUI theme if you don't pass in a custom theme.